cmd/monaco/integrationtest/v2/bucket_integration_test.go

```diff
@@ -0,0 +1,52 @@
+//go:build integration
+// +build integration
+
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/runner"
+	"github.com/spf13/afero"
+)
+
+// tests all configs for a single environment
+func TestIntegrationBucket(t *testing.T) {
+
+	configFolder := "test-resources/integration-bucket/"
+	manifest := configFolder + "manifest.yaml"
+	specificEnvironment := ""
+	t.Setenv("MONACO_FEAT_BUCKETS", "1")
+
+	RunIntegrationWithCleanup(t, configFolder, manifest, specificEnvironment, "Buckets", func(fs afero.Fs, _ TestContext) {
+
+		// This causes Creation of all automation objects
+		cmd := runner.BuildCli(fs)
+		cmd.SetArgs([]string{"deploy", "--verbose", manifest})
+		err := cmd.Execute()
+		assert.NoError(t, err)
+
+		// This causes an Update of all automation objects
+		cmd = runner.BuildCli(fs)
+		cmd.SetArgs([]string{"deploy", "--verbose", manifest})
+		err = cmd.Execute()
+		assert.NoError(t, err)
```

Comment:
for a future PR/final implementation: we should extend `integrationtest/v2/assert.go` to allow checking for bucket API types and extend the test to include the assertion

Reply:
Implemented